### ISSUE TITLE
Question about stable decission for entrypoint filename

### ISSUE THREAD

**Issue by Gruummy** [docker-library/postgres#1116]
Hi community,

for the reason about a concept planning on my side (for the project in my company where i work in) i would have a theoretically really easy to answer  .... but important question:

Can we as community trust on the situation that the entrypoint scipt of running postgres ... in dockerized worlds  ... will stay to be 
"/docker-entrypoint.sh" ?

I ask for multiple reasons. 2 of them have the following background:
- https://github.com/pgautoupgrade/docker-pgautoupgrade
- https://github.com/docker-library/postgres/issues/1094

I have a few years experiance with postgres ... so no fear that the do not understand the complexitiy of this RDBMS and such questions.
But we create concepts which should work for years ... in our company .. so this question is a very important one ... to ensure that not "surprisingly" the world changes over night.

It would be great if a reliable answer could be given.

... please excuse typos ... wrong ordering of words ... and so on. My native language is german and i am to lazy to put everything before ... what is going through my mind .... to google translator.

**Comment by yosifkit** [comment]
We have no plans to change the name of the file; the filename will stay `docker-entrypoint.sh`. It is at `/usr/local/bin/docker-entrypoint.sh` within the images.

It has not been available at `/` in the image since before PostgreSQL `11`.  I don't foresee us moving it away from `/usr/local/bin` but if we did, it would still be findable in the `PATH`. https://github.com/docker-library/postgres/pull/260 is where it was added to `PATH` instead of `/`, but with a symlink at `/`. Our template used to make the symlink to `/` for versions less than 11, but it was no longer being used since those versions have reached end of life:
```
{{ if .major >= 11 then "" else ( -}}
RUN ln -s usr/local/bin/docker-entrypoint.sh / # backwards compat
{{ ) end -}}
```
